src/create_disk.sh

```diff
@@ -60,13 +60,13 @@ grub2-install \
 	$disk
 
 # install uefi grub
-mkdir -p rootfs/boot/efi/EFI/boot
-grub2-mkimage \
-	--format x86_64-efi \
-	--output rootfs/boot/efi/EFI/boot/bootx64.efi \
-	--prefix '(hd0,gpt1)/grub2' \
-	normal fat part_gpt gzio terminal configfile echo ext2
-cp -r /usr/lib/grub/x86_64-efi rootfs/boot/grub2/
+mkdir -p rootfs/boot/efi/EFI/{BOOT,fedora}
+cp /boot/efi/EFI/BOOT/BOOTX64.EFI rootfs/boot/efi/EFI/BOOT/BOOTX64.EFI
```

Comment:
This will work because the firmware default boot behavior searches for `EFI/BOOT/BOOTX64.EFI` when the boot options aren't defined (there isn't a `BootOrder` and `Boot####` EFI variables).

But usually in that case what most OS that use shim do is to rely on shim's fallback mechanism to populate the boot options from a `BOOT.CSV`, which is a UCS-2 file of comma separated values that provides a list of boot entries to create. So I wonder if FCOS shouldn't do the same.

For example in traditional Fedora we have:

```
cat /boot/efi/EFI/fedora/BOOTX64.CSV 
shimx64.efi,Fedora,,This is the boot entry for Fedora
```

The EFI default boot behavior and shim's fallback mechanism is explained in detail [here](https://blog.uncooperative.org/blog/2014/02/06/the-efi-system-partition/).